pex/pex.py

```diff
@@ -155,9 +155,14 @@ def minimum_sys_modules(cls, site_libs, modules=None):
 
   @classmethod
   def minimum_sys_path(cls, site_libs, inherit_path):
+    ver = sys.version[:3]
     scrub_paths = OrderedSet()
     site_distributions = OrderedSet()
     user_site_distributions = OrderedSet()
+    undesirable_paths = OrderedSet([
+      '/System/Library/Frameworks/Python.framework/Versions/%s/Extras/lib/python' % ver,
```

Comment:
we should never ever hardcode paths like this into pex.  it sounds like PEX._extras_paths is not behaving the way as intended and should be fixed.  these paths are available programmatically:

```
=; PEX_INTERPRETER=1 PEX_IGNORE_RCFILES=1 PEX_PYTHON=/usr/bin/python2.7 pex
Python 2.7.5 (default, Mar  9 2014, 22:15:05) 
[GCC 4.2.1 Compatible Apple LLVM 5.0 (clang-500.0.68)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
(InteractiveConsole)
>>> from pex.pex import PEX
>>> for path in PEX._extras_paths():
...   print(path)
... 
/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/../../Extras/lib/python
```